### PR TITLE
Few more fixes and improvements to data we are collecting

### PR DESCRIPTION
**Pull request by jhutar**
I'm not sure about this `__version__` thing. What do you think @pmacik ?

Intention is to be able to change that field in scenario file whenever scenario significantly changes (e.g. new task is added) so performance of that scenario is expected to change.

It is slightly duplicated by a last git commit, but benefit of having it would be this version only changes when there is a significant change in scenario itself, not when somebody updates a readme or so.

### CODE REVIEW

File: ci-scripts/collect-results.sh [hunk 1]
```diff
@@ -30,11 +30,15 @@ set -u
 mstart=$(date --utc --date "$(cat benchmark-before)" --iso-8601=seconds)
 mend=$(date --utc --date "$(cat benchmark-after)" --iso-8601=seconds)
 mhost=$(kubectl -n openshift-monitoring get route -l app.kubernetes.io/name=thanos-query -o json | jq --raw-output '.items[0].spec.host')
+mversion=$(sed -n 's/^__version__ = "\(.*\)"/\1/p' scenarios/$(cat benchmark-scenario).py)
```
Comment (pmacik) [inline ci-scripts/collect-results.sh:33]: Why cannot you just use th `SCENARIO` env variable directly?
```suggestion
mversion=$(sed -n 's/^__version__ = "\(.*\)"/\1/p' scenarios/${SCENARIO}.py)
```

**Comment by pmacik** [comment]
@jhutar I like the idea of versioning scenarios

File: ci-scripts/collect-results.sh (hunk 1 above)
Comment (jhutar) [reply ci-scripts/collect-results.sh:33]: We only have that variable in `ci-scripts/test.sh`. Maybe if this becomes secret in Vault, we can take if from that file.

Comment (pmacik) [reply ci-scripts/collect-results.sh:33]: Actually, the `SCENARIO` env variable along with [others](https://github.com/openshift/release/blob/master/ci-operator/config/redhat-performance/backstage-performance/redhat-performance-backstage-performance-main.yaml#L50) is set by the CI and in the `ci-scripts/test.sh` is just overriden temporarily and hard-coded to the only rhdh scenario we currently have. So, you are correct that to make it work in the `collect-results.sh` we should override it similarly as in `test.sh` (for now).

Comment (pmacik) [reply ci-scripts/collect-results.sh:33]: Let's do that another time

**Pull request merged**